src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -227,8 +222,7 @@ public void testOnEmptyFile() throws Exception {
         final String pathToEmptyFile =
                 File.createTempFile("file", ".java", temporaryFolder).getPath();
 
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, pathToEmptyFile, expected);
+        execute(checkConfig, pathToEmptyFile);
```

Comment:
We can make assert on size of file to be zero, it will allow bypass pmd.

I don't know how to do validation on file that must be empty, so no header allowed.

Reply:
@romani done with assert on size. I found no way to make input file